### ISSUE TITLE
[fast-website] Form components are missing labels

### ISSUE THREAD

**Issue by chrisdholt** [microsoft/fast#3465]
**Describe the bug; what happened?**
On the website, many of the form components are missing labels. This is creating accessibility issues as users do not have context for those elements.

**What behavior did you expect?**
All form elements should have a corresponding label or an aria label to give it an accessible name.

To preserve the design integrity, we should likely add aria-labels to the form elements and any other elements missing accessible names in the "fast-frame" portion of the website.

**Comment by abhishekjakhar** [comment]
Can I pick this up?

**Comment by khamudom** [comment]
@abhishekjakhar, I am currently working on this now.
